template/module/INSTALL.rst

```diff
@@ -0,0 +1,7 @@
+[ this file must only be present if there are very specific
+  installation instructions, such as installing non-python
+  dependencies; the audience is systems administrators  ]
+
+To install this module, you need to:
+
+#. Do this ...
```

Comment:
Well I'm not sure yet. Since INSTALL and CONFIGURE are for different audiences (system administrators and advanced users respectively), they can end-up in completely different place in a more complete documentation, so we should be able to include them separately in that larger book. Same reason why I kept ROADMAP and DEVELOP separate for now.

Reply:
In general better to keep the options open for granularity.
We are talking about 10 files in total: it looks reasonable